pkg/validate/providers.go

```diff
@@ -0,0 +1,89 @@
+package validate
```

Comment:
Suggestion: rename this file from `providers.go` to `accounts.go`, since there are account types that are not a provider (jenkins, slack, etc.)

Reply:
So many `accounts.go` but makes sense.